<launch>
    <node name="uav_control" pkg="sunray_uav_control" type="uav_control_node" output="screen">
        <param name="uav_id" type="int" value="1" />
        <param name="uav_name" type="string" value="uav" />
        <!-- 起飞高度 降落第一阶段高度 降落速度 -->
        <param name="uav_control/Takeoff_height" value="0.6" />
        <param name="uav_control/Disarm_height" value="0.2" />
        <param name="uav_control/Land_speed" value="0.2" />
        <!-- 安全围栏 超出会降落 -->
        <param name="geo_fence/x_min" value="-10.0" />
        <param name="geo_fence/x_max" value="10.0" />
        <param name="geo_fence/y_min" value="-10.0" />
        <param name="geo_fence/y_max" value="10.0" />
        <param name="geo_fence/z_min" value="-1.0" />
        <param name="geo_fence/z_max" value="3.0" />
        <!-- 降落最后一阶段需要的时间和速度 -->
        <param name="flight_param/land_end_time" value="1.0" />
        <param name="land_end_speed" value="0.3" />
        <!-- 定位超时警告和降落时间 -->
        <param name="odom_valid_timeout" value="0.5" />
        <param name="odom_valid_warming_time" value="0.3" />
        <!-- 默认home点 -->
        <param name="default/home_x" value="0.0" />
        <param name="default/home_y" value="0.0" />
        <param name="default/home_z" value="0.0" />
        <!-- 是否使用遥控器控制 如果不使用则不允许进入RC_CONTROL模式，允许在解锁前切换到offboard模式再解锁-->
        <param name="use_rc_control" type="bool" value="true" />
        <!-- 是否检测指令超时 允许两条指令之间的时长 超时后进入悬停-->
        <param name="check_cmd_timeout" type="bool" value="false" />
        <param name="cmd_timeout" value="5.0" />
    </node>
</launch>
